core/org.openjdk.jmc.flightrecorder.rules.jdk/src/main/resources/org/openjdk/jmc/flightrecorder/rules/jdk/messages/internal/messages.properties

```diff
@@ -643,6 +643,11 @@ PasswordsInSystemPropertiesRule_TEXT_INFO_EXCLUDED_INFO=The following regular ex
 PasswordsInSystemPropertiesRule_TEXT_INFO_LONG=The following suspicious system properties were found in this recording: {suspiciousSystemProperties}.
 PasswordsInSystemPropertiesRule_TEXT_SOLUTION=If you wish to keep having passwords in your system properties, but want to be able to share recordings without also sharing the passwords, please disable the ''Initial System Property'' event.
 PasswordsInSystemPropertiesRule_TEXT_OK=The recording does not seem to contain passwords in the system properties.
+PID1Rule_RULE_NAME=PID-1 Rule
+PID1Rule_TEXT_OK=No Process runs with PID 1.
+PID1Rule_TEXT_INFO=Process runs with PID 1.
+PID1Rule_TEXT_INFO_LONG=Java processes shouldn't run with PID 1 as this PID is for the init process in Unix-like systems and receives special treatment.
```

Comment:
I think this explanatory text can be improved, more like what Marcus wrote in the issue tracking this.